packages/vite-plugin-svelte/src/utils/error.js

```diff
@@ -114,6 +114,7 @@ export function enhanceCompileError(err, originalCode, preprocessors) {
 
 	// Handle incorrect CSS preprocessor usage
 	if (err.code === 'css-syntax-error') {
```

Comment:
@dummdidumm  whats the new way to identify css syntax errors? i looked but didn't find the corrosponding code in svelte5

Reply:
added with generic startswith css_ , technically this could introduce false positives for users who don't use a css preprocessor, so if we want to play it conservative, we'd have to use a hardcoded list at the cost of that getting out of date if new codes are added.